cq-image/Makefile

```diff
@@ -0,0 +1,68 @@
+.PHONY := lint check fix docker_lint jinja_lint python_lint_check shell_lint yaml_lint jinja_format_check python_format_check shell_format_check yaml_format_check jinja_format_fix python_lint_fix python_format_fix shell_format_fix yaml_format_fix help
+
+all: lint check
+
+lint: docker_lint jinja_lint python_lint_check shell_lint yaml_lint
```

Comment:
The naming convention appears to be _something_._lint. So why do we have the extra check in `python_lint_check`?

Reply:
On line 36, we have the following:
```makefile

python_lint_check:
	-ruff check /code --output-format=github

python_lint_fix:
	ruff check /code --fix --output-format=github
```

My intention with the interface is to force users to be explicit when they want to change something. Since `ruff` provides the ability to fix linting errors, I separated the `python_lint` command into two parts, one to analyze the code and one to fix issues (which aligns with the rest of the interface).

Reply:
Wouldn't `python_lint_check` then be instead on line 7?

Reply:
`python_lint_check` is a member of the `lint` group because it's a linting command. Formatting commands are covered by the `check` group on line 7.

I was a bit conflicted on the naming for `python_lint_check`, because it does adopt the `check`->`fix` pattern used by the formatting commands. Maybe it would be better to rename `python_lint_check` to `python_lint`, and remove `python_lint_fix` from the `fix` group, since it doesn't fix formatting (like all the other `fix` commands).